### PR TITLE
Readme

### LINKED ISSUE #103: Readme for the extension
In the readme we would cover:
- how to run the extension
- how to report a feedback
- Link on how to extend the catalog (#41)

### PR DESCRIPTION
**Pull request by feloy**
fixes #103 

- [x] how to run the extension
- [x] how to report a feedback
- [x] how to extend the catalog

### CODE REVIEW

**Comment by feloy** [comment]
@slemeur do you have more info on how we want users to send feedback?

File: README.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
...
+
+From the Podman Desktop sources folder:
+
+```
+$ yarn watch --extension-folder path-to-extension-sources-folder/packages/backend
+```
+
+## Providing a custom catalog
+
+The extension provides a default catalog, but you can build your own catalog by creating a file `$HOME/podman-desktop/ai-studio/catalog.json`.
```
Comment (slemeur) [inline README.md:23]: maybe we can extend by showing where the file is in the source code  so the user knows how to conform to the format and mentioning to do a PR.

**Comment by slemeur** [comment]
Sounds a good start for me.
For the feedback, I think we should probably have a gform setup, something quick, but I'm thinking that we would like to collect information about the following:
- Information about the setup: OS, CPU, GPU, podman machine setup
- Information about the model used
- Free form section so the user can provide the feedback
- email, so we can ask more.
- rating, this way we can get a sense of how people evaluate the effort

**Comment by slemeur** [comment]
cc @mairin

**Comment by mairin** [comment]
**Operating System**
[ macOS, Windows, Linux, other]

**If Linux:**
[ Fedora, RHEL, CentOS Stream, Debian, Ubuntu, Arch, SuSE, Mint, Pop!OS, ElementaryOS, other ]

**CPU**
[ AMD/Intel 64, ARM 64, other]

**GPU**
[ None, Apple, Nvidia, Intel, AMD, other]

**Podman Version**
// need easy instructions to look up
[ v4.8.3 (3 Jan 2024), v4.8.2 (11 Dec 2023), v4.8.1 (5 Dec 2023), v4.8.0 (27 Nov 2023), v4.7.2 (31 Oct 2023), v4.7.1 (5 Oct 2023), v4.7.0 (27 Sep 2023), v4.6.x (Jul-Aug 2023), v4.5.x (Apr-Jun 2023), v4.4.x (Feb-Mar 2023), v4.3.x (Oct 2022 - Jan 2023), older, not sure]

**Podman Machine**
[ qemu (default), vfkit, not sure ]

**Model Used**
[ Mistral | Mixtral | llama2 | other]

If other: [ // freeform ]

**Give us your feedback:**

[ // freeform field ]

**Your Rating**

[ 5 - excellent, 4 - good, 3 - ok, 2 - meh, 1 - not good]

File: README.md (hunk 1 above)
Comment (benoitf) [reply README.md:23]: it's not related to the readme but probably folder should not contain the name podman-desktop as Podman Desktop is not owning it, it's an extension choice.

**Review by benoitf — APPROVED** [review]
maybe time to merge and iterate

File: README.md (hunk 1 above)
Comment (benoitf) [reply README.md:23]: --> `$HOME/ai-studio`

**Pull request merged**